docs/advanced/analytics/plugins.mdx

```diff
@@ -0,0 +1,340 @@
+---
+sidebar_position: 2
+title: Plugins
+description:
+  The analytics library is designed to work with any third party analytics tools
+  via plugins.
+---
+
+import AnalyticsFigure from "./assets/analytics.svg";
+import Figure from "@site/src/components/Figure";
+import ContactLink from "@site/src/components/ContactLink";
+import { useState } from "react";
+
+> _Since version 2.19_
+
+<p>{frontMatter.description}</p>
+
+Plugins are a powerful abstraction that let you:
+
+- add a new analytics provider (like google analytics or pixel)
```

Comment:
**question:** is "pixel" the name of "Meta pixel"? I think it isn't (but really not sure) and it  seems weird in this sentence.

**suggestion:** for consistency, rename with the exact service name
```suggestion
- add a new analytics provider (like Google analytics or Meta pixel)
```